### DISCUSSION TITLE
Retrieving data from nested dataset in scripted data source

### DISCUSSION THREAD

**Discussion by ndhasb** [eclipse-birt/birt#1937] [Q&A]
Hello,

I’m using a scripted data source for my report.

In the Report Parameter, I have these three values, which I retrieve from my repository.
![image](https://github.com/user-attachments/assets/aa9f4591-a4d5-462b-9258-f15d800a47d2)

In the event handler, this is the code I’ve applied:
![image](https://github.com/user-attachments/assets/82a04362-f900-4460-a8ca-fd76c7e0195d)

In the dataset parameter, here’s what I’m currently using:
![image](https://github.com/user-attachments/assets/d86fe5a7-42d8-4228-af12-3b367c4e4cc8)

In the nested dataset parameter, here's what I’m using. There’s an additional parameter called subFieldName, which is used to retrieve the nested dataset.
![image](https://github.com/user-attachments/assets/db952e76-0899-4db4-918d-aa9c5eb9e433)

This is the configuration I’ve set up to retrieve data from our repository. The dataset contains another nested dataset. However, I’m currently encountering an issue with fetching data from the nested dataset using these settings.

Does anyone have any suggestions or ideas on how to resolve this?

**Comment by hvbtup** [comment]
You don't show your code and you don't even tell what's the issue exactly.
The only information that you give us is like "my code does not work, but I can't show it to you."
You would curse your users if they gave you such unspecific error reports...
